Dockerfile

```
@@ -38,7 +38,10 @@ COPY ${SCOPE_CONFIG:-config.json} /app/config.json
 
```

In the beginning of the Dockerfile:
```suggestion
FROM debian:latest

ARG SCOPE_PORT=8080
```

---

Another concern about building/serving the asset from the container is with this hardcoded argument you can't change the desired port you want to run the app on without changing this tracked file and rebuilding the image